src/colours.cc

```diff
@@ -194,9 +194,11 @@ ColourRandomizer::Initialise ()
     if (!config.enabled)
         return;
 
-    RegisterHooks ({{HOOK_CALL, 0x5B8F17, (void *) &RandomizeColourTables},
-                    {HOOK_JUMP, 0x4C8500, (void *) &RandomizeVehicleColour}});
-
+    RegisterHooks (
```

Comment:
This hook is also related to the vehicle colour randomizer, so should also be a part of the if condition for car colours.

Reply:
Sorry for my ignorance, I am not sure how this could be taken over.